### DISCUSSION TITLE
Ecliptic obliquity - does it affect the reliability of the software.

### DISCUSSION THREAD

**Discussion by EkaSanatani** [Stellarium/stellarium#3654] [Q&A] [answered]
I am a member of a team doing research on Indian ancient history.
There is a debate about a certain event that is mentioned in one of our ancient scripts.
According to our hypothesis, that event which is mentioned should have not occured, but Stellarium software shows it occurred at certain time duration (~11K BCE to ~5K BCE). 

One of my team members made a statement that it was because of 'ecliptic obliquity'. She mentioned that in all planetary simulators, there is an 'assumption' that the earth's plane is tilted by 2 degrees. Periodically, the earth's plane will be moving up and down (obliquity) for 2 degrees. But this 2° obliquity is not yet proven. It's just an hypothesis. The software makers incorporate it. If they remove the obliquity factor, the software would be better.

What is your comment on this statement? 
Thanks

**Comment by gzotti** [comment, marked as answer]
The international community of astronomers has developed the current model for ecliptic obliquity. The model currently implemented has been published in 2011, with a minor correction in 2012. It is described to be valid for &pm; 200.000 years. We have no reason to doubt that.

Of course, for good planet positions before 4000BC you need the JPL DE441ephemeris file as described in the User Guide. 
Read Appendix F of the User Guide before answering: what does Stellarium show that should not have occurred? 

I don't know what "tilted by 2 degrees" refers to. Tilted against what? If we remove the periodic changes in ecliptic obliquity, the software becomes worse.

**Reply by EkaSanatani** [reply]
Thanks for the reply Sir. Will check back with my members and get back to Sir / Forum. Thanks again and have a nice day.
